mava/systems/jax/builder.py

```diff
@@ -197,7 +197,11 @@ def trainer(
         )
 
     def build(self) -> None:
-        """Construct program nodes."""
+        """Construct program nodes.
+
+        Returns:
+            None.
+        """
```

Comment:
I didn't bother adding `Returns: None` to init functions, because they always return none. Wherever I saw a function that could be interpreted as returning something, I added a `Returns: None`. Please feel free to flag any you think I missed.

Reply:
It is a minor thing no need to dive into it!